OC/config.plist

```diff
@@ -469,13 +495,41 @@
 		<key>Quirks</key>
```

Comment:
According to dortania's [tutorial](https://dortania.github.io/OpenCore-Install-Guide/config-laptop.plist/kaby-lake.html#kernel), you have to:
- disable `AppleCpuPmCfgLock`, `XhciPortLimit` (don't rely on it, says the tutorial. We already have DSL patch) 

However the effect is not tested yet.

Reply:
it seems no different. guess should change to false for this i guess. since we already have DSL patch